skimage/measure/_regionprops.py

```diff
@@ -627,25 +626,27 @@ def perimeter(image, neighbourhood=4):
     return total_perimeter
 
 
-def _parse_docs():
-    import re
-    import textwrap
+if __debug__:
+
+    def _parse_docs():
+        import re
+        import textwrap
 
-    doc = regionprops.__doc__
-    matches = re.finditer('\*\*(\w+)\*\* \:.*?\n(.*?)(?=\n    [\*\S]+)',
-                          doc, flags=re.DOTALL)
-    prop_doc = dict((m.group(1), textwrap.dedent(m.group(2))) for m in matches)
+        doc = regionprops.__doc__
+        matches = re.finditer('\*\*(\w+)\*\* \:.*?\n(.*?)(?=\n    [\*\S]+)',
+                              doc, flags=re.DOTALL)
+        prop_doc = dict((m.group(1), textwrap.dedent(m.group(2))) for m in matches)
 
-    return prop_doc
+        return prop_doc
 
 
-def _install_properties_docs():
-    prop_doc = _parse_docs()
+    def _install_properties_docs():
+        prop_doc = _parse_docs()
 
-    for p in [member for member in dir(_RegionProperties)
-              if not member.startswith('_')]:
-        getattr(_RegionProperties, p).__doc__ = prop_doc[p]
-        setattr(_RegionProperties, p, property(getattr(_RegionProperties, p)))
+        for p in [member for member in dir(_RegionProperties)
+                  if not member.startswith('_')]:
+            getattr(_RegionProperties, p).__doc__ = prop_doc[p]
+            setattr(_RegionProperties, p, property(getattr(_RegionProperties, p)))
 
 
-_install_properties_docs()
+    _install_properties_docs()
```

Comment:
If this PR fixes your issue, 
It seems like all you need to do is have this one line in the __debug__.

Reply:
Am I missing something?

Reply:
nothing missed.

I was hesitating to keep or not keep these 2 related functions in the path of the non-debug mode..

I finally choosed not to keep them but if you prefer to keep the 2 functions defined when they are unused then please acknowledge so.. if you prefer the simpler one liner fix so.

for the test : given by default tests are executed without -OO I've done nothing here.
but it's something pytest handles correctly as far as I see.

Reply:
yeah, so I would remove the indentation. those functions may want to be called otherwise.